### DISCUSSION TITLE
能不能支持特殊EFI文件的启动？比如黑苹果用到的OpenCore、Clover之类的

### DISCUSSION THREAD

**Discussion by gujiangjiang** [ventoy/Ventoy#2067] [Ideas]
能不能支持特殊EFI文件的启动？比如黑苹果用到的OpenCore、Clover之类的

现在的Ventory也支持一些常用的efi文件的启动，比如shell.efi之类的，作为长期黑果用户，如果可以支持引导跳转到OP或者Clover的话那就很神奇了，我自己测试过不行，可能跟路径有关系，想了一下很早以前Clover提供iso文件的镜像是可以启动的，我想可不可以运用类似于文件注入的方式，比如创建一个文件夹OpenCore，然后提供一个OpenCore.img的空镜像文件，然后选择OpenCore.img启动的时候将文件夹OpenCore里的所有文件全部注入到OpenCore.img里，然后就可以正常启动了，能力有限，未经测试，感觉应该是可行的，不过OP和Clover可能牵扯到EFI驱动的顺序和关联情况，具体的我也不太懂，只能让大佬看看可不可行了，如果支持的话又可以省一个优盘了。

PS：OpenCore不仅仅适用于黑苹果，也适用于白苹果。Clover不仅仅适用于黑苹果，也适用于普通PC，还支持Legacy的电脑以UEFI的方式安装系统，所以这两个工具还是具有一定的可用性的。

**Comment by ventoy** [comment]
这种最好是安装Ventoy时保留一部分空间，然后创建第3个分区，把这个分区格式化为FAT32，作为EFI分区。然后在这个分区内搞OC。
https://www.ventoy.net/cn/doc_disk_layout.html#reserve_space
